modules/ROOT/pages/configure-notifications.adoc

```diff
@@ -11,7 +11,7 @@ Each email notifications includes:
 [[send-auto-notifs]]
 == Send Automated Conformance Notifications
 
-To automatically send notifications when the latest version of an API becomes non-conformant, enable and configure the notifications. 
+Enable and configure the notifications to automatically send notifications when the latest version of an API becomes non-conformant. 
```

Comment:
notifications notifications - any way to use a different word?

Reply:
For now I can change it to email. We'll have to update it later when they send other kinds, but we'll know well in advance.